virtualcluster/pkg/vn-agent/server/metrics.go

```diff
@@ -0,0 +1,124 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```

Comment:
Can you update the copyright on this to 2022.

```suggestion
Copyright 2019 The Kubernetes Authors.
```